.github/workflows/tests.yaml

```diff
@@ -118,6 +124,12 @@ jobs:
         with:
           name: image diff output directory
           path: browser-test/diff_output
+      - name: Upload test videos on failure
+        uses: actions/upload-artifact@v3
```

Comment:
Just one question throguh- where will the videos be uploaded to? Will it be in github?

Reply:
Yes, when a test fails it will upload videos and they will be available on "Summary" page of the github action that failed. Here is the example: https://github.com/civiform/civiform/actions/runs/2886074984 (scroll all the way down).